clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -356,8 +357,7 @@ public void verifyHeartbeatSent() throws Exception {
         // We need two update calls:
         // 1. the first call "sends" the metadata update requests
         // 2. the second one gets the response we already queued up
-        consumer.updateAssignmentMetadataIfNeeded(0L);
-        consumer.updateAssignmentMetadataIfNeeded(0L);
+        consumer.updateAssignmentMetadataIfNeeded(time.timer(Long.MAX_VALUE));
```

Comment:
Seems like we need to update the comment above. Also, why are we replacing the two calls by a single call everywhere in this test?

Reply:
I removed them because we could get the same effect with a single call and a non-zero timeout. Ack on removing the comment.